documentation/api/wire_format/replace_catalog_inputs_format_v1.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: "PuppetDB: Replace catalog inputs wire format, version 1 (experimental)"
+layout: default
+canonical: "/puppetdb/latest/api/wire_format/replace_catalog_inputs_v1.html"
+---
+
+The v1 `replace catalog inputs` command tells PuppetDB to replace the current
+set of catalog inputs with the new.
+
+Changing the value of this setting for a certname has the same effect
+as receipt of a new catalog with respect to extending the lifetime of
+the node.
+
+Replace catalog inputs command format
+-----
+
+### Version
+
+This is **version 1** of the replace catalog inputs command.
+
+### Encoding
+
+The command is serialized as JSON, which requires strict UTF-8 encoding.
+
+### Main data type: Replace catalog inputs for node
+
+     {
+      "certname": <string>,
+      "producer_timestamp": <datetime>,
+      "catalog_uuid": <string>,
+      "inputs": [[<string>, <string>]],
```

Comment:
Let's add the ... to match our other docs, e.g.: `"inputs": [[<string>, <string>], ...]`